### ISSUE TITLE
Usability on small screens review

### ISSUE THREAD

**Issue by KatieWoe** [phetsims/beers-law-lab#236]
Per @kathy-phet's request in https://github.com/phetsims/QA/issues/295 I am making issues in each published sim repo detailing the pointer areas and usability of sims on phone sized screens.  Done on an iPhone 7 iOS 12 device.

Potential Difficulties Noticed:
* Radio buttons
* Left and right arrow buttons
* Blue pointer area of concentration tools looks off in landscape

Screenshots:
![Image from iOS (1)](https://user-images.githubusercontent.com/41024075/55906488-9e8df600-5b91-11e9-98be-5d93a14ebb32.png)
![Image from iOS (2)](https://user-images.githubusercontent.com/41024075/55906490-9e8df600-5b91-11e9-888e-2e033e5942a6.png)
![Image from iOS (3)](https://user-images.githubusercontent.com/41024075/55906491-9e8df600-5b91-11e9-8bf8-2ca6a279132e.png)
![Image from iOS](https://user-images.githubusercontent.com/41024075/55906492-9e8df600-5b91-11e9-9e7a-1526a1498435.png)

**Comment by pixelzoom** [comment]
@KatieWoe noted:
> Blue pointer area of concentration tools looks off in landscape

Please create a separate issue for that, it's likely a bug.

**Comment by arouinfar** [comment]
>Radio buttons

Let's include the solid/solution and preset/variable text in the touch area for the radio buttons.

>Left and right arrow buttons

The tweakers are in close proximity to other UI components, so we'd need to dilate the touch areas asymmetrically. For Wavelength, I think we would keep the same top/inner boundaries, and dilate a bit to the bottom/outside. For Concentration, I'd keep the same inner boundary, but dilate in the other 3 directions.

@pixelzoom here's a mockup with proposed touch areas in green. For reference, the tweaker touch areas are all the same size, but shifted to accommodate neighboring UI components.
![image](https://user-images.githubusercontent.com/8419308/55961870-5de7b880-5c35-11e9-9c60-80528ab6fc80.png)

 > Blue pointer area of concentration tools looks off in landscape

@KatieWoe please move this to a new issue in the `beers-law-lab` repo.

**Comment by KatieWoe** [comment]
New issue made

**Comment by pixelzoom** [comment]
@arouinfar wrote:
> > Radio buttons
> Let's include the solid/solution and preset/variable text in the touch area for the radio buttons.

The label on radio buttons (text, icon, or text + icon) are automatically included in default pointer areas. And default point areas are not displayed using `?showPointerAreas`.   So I question where anything needs to be done for the solid/solution buttons.  @arouinfar?

**Comment by pixelzoom** [comment]
@arouinfar said:
> For Wavelength, I think we would keep the same top/inner boundaries, and dilate a bit to the bottom/outside.

This is not supported by the WavelengthSlider common-code component.  Horizontal dilation is indeed to the outside direction.  But top and bottom dilation must be the same. So I'm going to proceed with that.

**Comment by pixelzoom** [comment]
@arouinfar wrote:
> For Concentration, I'd keep the same inner boundary, but dilate in the other 3 directions.

Concentration is a NumberControl, and this is currently not supported.  In fact, this seems to be a general problem with NumberControl.  Pointer areas for the arrow buttons can only be dilated via `arrowButtonOptions`, and the dilation is the same amount around the button. That results in overlap with the slider thumb, e.g. in the screenshot below.   What we want is dilation similar to the arrow buttons for WavelengthSlider.  I'll create a scenery-phet issue to address this.

![screenshot_1130](https://user-images.githubusercontent.com/3046552/56009139-fb75d300-5c9b-11e9-9e45-71f475f1648e.jpg)

**Comment by pixelzoom** [comment]
List of UI elements to be adjusted:

- [ ] Solid and Solution radio buttons
- [ ] preset and variable radio buttons
- [ ] wavelength arrow buttons
- [ ] Concentration arrow buttons (phetsims/scenery-phet#236)
- [ ] Concentration probe (https://github.com/phetsims/scenery-phet/issues/488)

**Comment by pixelzoom** [comment]
Blocked by https://github.com/phetsims/scenery-phet/issues/488.

**Comment by pixelzoom** [comment]
4/25/19 dev meeting:

@ariel-phet said that the blocking issue phetsims/scenery-phet#488 is not a priority, so neither is this issue.  Unassigning.

**Comment by pixelzoom** [comment]
I moved the ProbeNode issue to https://github.com/phetsims/beers-law-lab/issues/239.  It was the last issue discussed herein, and it's unlikely to be addressed in the near future.  Closing this issue.
